production/helm/templates/loki/deployment.yaml

```diff
@@ -53,33 +51,21 @@ spec:
               mountPath: "/data"
               subPath: {{ .Values.loki.persistence.subPath }}
           ports:
-            - name: {{ template "loki.name" . }}-metrics
-              containerPort: {{ .Values.loki.service.port }}
+            - name: http
```

Comment:
We're using `-metrics` suffix as a marker for prometheus service discovery. Any downside in keeping this for deployments?

Reply:
@davkal The name isn't important, it just needs to be a static value (i.e. not templated) so it can be used as named ports for the probes in values.yaml. Changed it to `http-metrics`, which also matches the existing promtail port name. Does that resolve your concern?